lib/glimesh/api/schema/channel_types.ex

```diff
@@ -106,6 +106,14 @@ defmodule Glimesh.Api.ChannelTypes do
 
       resolve(&ChannelResolver.upload_stream_thumbnail/3)
     end
+
+    @desc "Update a channel's title"
+    field :change_title, type: :channel do
+      arg(:channel_id, non_null(:id))
+      arg(:title, non_null(:string))
+
+      resolve(&ChannelResolver.change_title/3)
```

Comment:
In respect to the other comment I made, this could be `update_channel` / `update_channel_info` or something similar, to allow us easier access to add new changeable fields.